src/js/core/widget/core/scroller/effect/Bouncing.js

```diff
@@ -61,7 +62,8 @@
 				},
 				Orientation = {
 					VERTICAL: "vertical",
-					HORIZONTAL: "horizontal"
+					HORIZONTAL: "horizontal",
+					VERTICALANDHORIZONTAL: "verticalAndHorizontal"
```

Comment:
This property name makes some confusion... maybe: VERTICAL_HORIZONTAL

Reply:
Changed to VERTICAL_HORIZONTAL
Plus changed property value to: vertical-horizontal